ext/odbc/tests/odbc_free_result_001.phpt

```diff
@@ -11,15 +11,13 @@ include 'config.inc';
 
 $conn = odbc_connect($dsn, $user, $pass);
 
-odbc_exec($conn, 'CREATE DATABASE odbcTEST');
+odbc_exec($conn, 'CREATE TABLE free_result (TEST INT NOT NULL)');
+odbc_exec($conn, 'ALTER TABLE free_result ADD PRIMARY KEY (TEST)');
```

Comment:
```suggestion
odbc_exec($conn, 'CREATE TABLE free_result (TEST INT PRIMARY KEY NOT NULL)');
```
Doesn't that do the same?

Reply:
Yeah, sure! I'm not sure why I used this syntax 🤔 btw. setting the PK is not even necessary here..